package.json

```diff
@@ -42,6 +52,7 @@
     "raw-loader": "^3.1.0",
     "react": "^16.8.4",
     "react-axe": "^3.0.2",
+    "react-docgen-typescript-loader": "^3.2.1",
```

Comment:
Maybe I'm overlooking it, but where do we benefit from this package? Is it supposed to generate prop tables below the storybook entries?

Reply:
Yes, I'll update stories to make use of it.